packages/vuetify/src/components/VNavigationDrawer/VNavigationDrawer.ts

```diff
@@ -351,6 +351,8 @@ export default baseMixins.extend({
     },
     genListeners () {
       const on: Record<string, (e: Event) => void> = {
+        mouseenter: () => (this.isMouseover = true),
+        mouseleave: () => (this.isMouseover = false),
```

Comment:
Is it possible that these two lines should be added only `if (this.expandOnHover)`, like it was [before](https://github.com/vuetifyjs/vuetify/pull/14523/files#diff-98d92001e9cf0f207f45c674457cea37a833979a1b0f0b143cb3714454acdddeL369)?